### ISSUE TITLE
Outgoing service calls that is configurable on a service by service basis 

### ISSUE THREAD

**Issue by kvbanantha** [apiman/apiman#725]
Can I configure proxy server with username and passwords for outgoing requests? I tried to find proxy configurations in the admin console, could not find any configuration or policy related to proxy. Do I need to implement a policy for that?

**Comment by volkflo** [comment]
Do you mean something like this?
![image](https://user-images.githubusercontent.com/13332383/55620824-3196ec80-579c-11e9-83d8-fee95aca3af0.png)
Under the **Implementation** tab for each API

**Comment by kvbanantha** [comment]
Yes, Some thing like that, but not for basic authentication. I am looking for Proxy Username Password to apply for outgoing service requests. so, each out going service request will go through the proxy server and authentication happens using the username/password settings

**Comment by EricWittmann** [comment]
You are looking for different username/password proxy settings for each outgoing service request?  Typically proxy settings would be configured globally for the gateway.  I've not  heard of a use-case where different proxy auth creds are used depending on the outbound destination.
